src/polycubed/src/server/Resources/Endpoint/ListResource.cpp

```diff
@@ -19,6 +19,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <../../config.h>
```

Comment:
I would move this one line down, to keep an empty line between standard headers and the ones of the project.